### ISSUE TITLE
"dnvm use" doesn't make runtime active when open new cmd

### ISSUE THREAD

**Issue by zoranpro** [aspnet/dnvm#262]
Hi,
I am experiencing the issue with not having active runtime when I open new cmd. If I do `dnvm list` I see:
Active Version     Runtime Architecture Location                       Alias
       1.0.0-beta4 clr     x86          C:\Users\zprotic\.dnx\runtimes default

After I do `dnvm use default` then this runtime is set to be active one. When I open new cmd and do again `dnvm list` I again don't have active runtime.

**Comment by davidfowl** [comment]
Use `dnvm use -p` to make it persist across instances.

**Comment by zoranpro** [comment]
Tried it. Didn't help. Any other suggestion?

**Comment by davidfowl** [comment]
What was the output when you tried it?

**Comment by zoranpro** [comment]
dnvm use default -p
Adding C:\Users\zprotic\.dnx\runtimes\dnx-clr-win-x86.1.0.0-beta4\bin to process PATH
Adding C:\Users\zprotic\.dnx\runtimes\dnx-clr-win-x86.1.0.0-beta4\bin to user PATH

**Comment by davidfowl** [comment]
And when you open a new shell you have no active dnx?

**Comment by zoranpro** [comment]
Exactly.

**Comment by davidfowl** [comment]
What does your PATH environment variable look like on the new shell

**Comment by zoranpro** [comment]
It does have only this: C:\Users\zprotic\.dnx\bin. In the one that is working I have this one as well: C:\Users\zprotic\.dnx\runtimes\dnx-clr-win-x86.1.0.0-beta4\bin;
When I checked in System Properties for user variables I see I have it: C:\Users\zprotic\.dnx\runtimes\dnx-clr-win-x86.1.0.0-beta4\bin;

**Comment by davidfowl** [comment]
So every brand new shell has that broken PATH?

**Comment by zoranpro** [comment]
Yes. Tried the same process several times.

**Comment by zoranpro** [comment]
One thing that might be helpful. When I installed DNVM from the powershell script provided on the http://docs.asp.net/ I experienced the same issue with new cmd. When I tried to type dnvm on new cmd it wasn't recognized. I solved this issue by manually adding C:\Users\zprotic\.dnx\bin; into System Variables. Was that the proper way to do it?

**Comment by glennc** [comment]
What OS is this? Are you running it from Powershell or CMD?

**Comment by zoranpro** [comment]
Sorry for late response. I was using CMD and Windows7 x64. That was happening before I have installed VS2015 RC. After I installed the VS, now is working as expected.

**Comment by glennc** [comment]
I am going to go with this being a bug that we resolved in a different fix, since it started working. If we see  it reappear then we can track it down and fix it then.

**Comment by glatzert** [comment]
I am experiencing this Problem, too.
Win10, Domain joined, VS 2015.1, ASP.NET RC1

**Comment by cda-1** [comment]
I experienced this problem as well, but found the issue. 

While using the -p switch didn't seem to allow my selection to survive the instance of cmd, I noticed that this was only because I was launching new instances of cmd my shift-right-clicking in an instance of windows explorer and launching the command window from there. I guess it's somehow linked to the old path/environment when doing that.

Running cmd.exe from the run dialog yielded an instance that could see the correctly persisted runtime version.

So I guess explorer.exe "hangs on" to an old environment. I suspect a new instance of explorer.exe would see the new environment!
